File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.lower()
        k = []
        s = s.split()
        for i in s:
            for j in range(len(i)):
                if i[j].isalnum():
                    k.append(i[j])
        k = "".join(k)
        p = list(k)
        p =p[::-1]
        p = "".join(p)
        return  p == k

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string new_s = "";
        for(char ch : s){
            if(isalnum(ch)){
                new_s += tolower(ch) ;
            }
        }
        int i = 0, j =new_s.size()-1 ; 
        while(i<j){
            if(new_s[i]==new_s[j]){
                i++;
                j-- ;
            }
            else{
                return false ;
            }
        }
        return true ; 
        
    }
};